### ISSUE TITLE
Re-add blog?

### ISSUE THREAD

**Issue by evanharmon1** [AdmiralFraggle/sommer-lawn#21]
Current theme also has good blog support

**Comment by AdmiralFraggle** [comment]
Yep we should add a blog. never worked right last time. The pages/articles worked fine but the search feature and blog page never worked right.
